src/pages/DelegationCreatePage/DelegationCreate.page.tsx

```diff
@@ -0,0 +1,82 @@
+import { PageContainer, SectionContainer } from '@/components/layout/containers'
+import { useNavigate } from '@/router'
+import { Stack } from '@mui/material'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { DelegationCreateCards } from './components/DelegationCreateCards'
+import { StepActions } from '@/components/shared/StepActions'
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward'
+import { match } from 'ts-pattern'
+import { DelegationCreateForm } from './components/DelegationCreateForm'
+
+/***
+ * It shows the cards component to choose delegation kind or the form component based on the state KIND or FORM
+ */
+export const DelegationCreatePage: React.FC = () => {
+  const { t } = useTranslation('party')
+  const navigate = useNavigate()
+  const [delegationKind, setDelegationKind] = useState<'CONSUME' | 'PROVIDE' | ''>('')
+  const [activeStep, setActiveStep] = useState<'KIND' | 'FORM'>('KIND')
+
+  return (
+    <PageContainer
+      title={t('delegations.create.title')}
+      backToAction={{
+        label: t('delegations.actions.backToTenant'),
+        to: 'TENANT',
+      }}
+    >
+      <SectionContainer
+        title={match({ activeStep, delegationKind })
+          .with({ activeStep: 'KIND' }, () => t('delegations.create.kindSectionTitle'))
+          .with({ activeStep: 'FORM', delegationKind: 'PROVIDE' }, () =>
+            t('delegations.create.provideDelegationTitle')
+          )
+          .otherwise(() => t('delegations.create.consumeDelegationTitle'))}
+      >
```

Comment:
Can you extract this logic in a constant outside the jsx? 
Something like:

`const sectionLabel = ...`